src/app/modules/item/pages/item-edit/item-edit.component.ts

```diff
@@ -75,7 +75,10 @@ export class ItemEditComponent implements OnDestroy {
   }
 
   saveInput() {
-    if (this.itemForm.invalid) return;
+    if (this.itemForm.invalid) {
+      this.errorToast('The form isn\'t valid');
```

Comment:
"You need to solve all the errors displayed in the form to save changes."

Reply:
Done